src/main/java/frc/robot/Robot.java

```diff
@@ -95,48 +136,70 @@ public void testInit() {
 
 
   }
-
+  @Override
   public void testPeriodic(){
 
   }
 
-
+  @Override
   public void disabledInit() {
-
+    if(Config.getInstance().getBoolean(Key.ROBOT__HAS_TURRET)){
+      mPotHandler.disable();
+    }
+    Config.getInstance().reload();
   }
-
+  @Override
   public void disabledPeriodic(){
-
+    if(Config.getInstance().getBoolean(Key.ROBOT__HAS_TURRET)){
+      sRobotLogger.verbose("pot " + Robot.mPot.get());
+    }
   }
 
 
   /*  
     Called constantly, houses the main functionality of robot
   */
   public void RobotLoop(){
-    //Driver Controls
-    //Check User Inputs
-    double DriveThrottle = mOperatorInterface.getDriveThrottle();
-    double DriveTurn = mOperatorInterface.getDriveTurn();
-    boolean AutoDrive = false;
-
-    //Continue Driving 
-    if(AutoDrive == true){
-      //AutoSteer Functionality
-      //Used for tracking a ball
-    }else{
-      //Standard Manual Drive
-      mDrive.setDrive(DriveThrottle, DriveTurn, false);
+    if(Config.getInstance().getBoolean(Key.ROBOT__HAS_TURRET)){
```

Comment:
Let's split the subsystems into their own functions.

Reply:
Fixed